Dockerfile.vncless

```
@@ -8,9 +8,11 @@ RUN if [ "${BUILD_ENV}" = "local" ]; then sed -i s/deb.debian.org/mirrors.aliyun
 	libgtk2.0-0 libx11-xcb1 libxtst6 libnss3 libasound2 libdbus-glib-1-2 iptables \
 	dante-server psmisc libxaw7 xclip busybox libssl-dev iproute2 tinyproxy-bin
 
+RUN apt-get install fonts-wqy-microhei
+
 RUN groupadd -r socks && useradd -r -g socks socks
 
-RUN cd tmp && apt update && apt download x11-utils && dpkg -x x11-utils_*.deb x11-utils &&\
+RUN cd tmp && apt-get download x11-utils && dpkg -x x11-utils_*.deb x11-utils &&\
```

```suggestion
RUN cd tmp && apt-get update && apt-get download x11-utils && dpkg -x x11-utils_*.deb x11-utils &&\
```
此处再次 `update` 的目的是防止复用前面的构建步骤时 apt 本地缓存已经过期

---

已在 4d3daa6dbd7b310cf31c83651a407dc3d0ce396d 中修改，感谢！